heartbeat/monitors/active/http/http_test.go

```diff
@@ -65,7 +65,7 @@ func testTLSRequest(t *testing.T, testURL string, extraConfig map[string]interfa
 	jobs, endpoints, err := create("tls", config)
 	require.NoError(t, err)
 
-	job := jobs[0]
+	job := monitors.WrapCommon(jobs, "tls", "", "http")[0]
```

Comment:
Ideally we wouldn't have this in tests, and the call to `create` above would be a different call that would do the wrapping. Unfortunately `newMonitor`, where this happens now would need a refactor for this to be clean. This PR has enough scope, so I'd like to defer that refactor.